plugin/core/views.py

```diff
@@ -479,8 +479,9 @@ def text2html(content: str) -> str:
     return re.sub(REPLACEMENT_RE, _replace_match, content)
 
 
-def make_link(href: str, text: str, class_name: Optional[str] = None) -> str:
-    text = text.replace(' ', '&nbsp;')
+def make_link(href: str, text: Any, class_name: Optional[str] = None) -> str:
```

Comment:
I think it is safe to typehint `text` as `str` instead of `Any`,
or declare a type `DocumentUri = str`. What do you think?

Reply:
The `diagnostic.code` is also passed into make_link, which might be an `int` (or `None`). But really, any stringifiable type works since it's passed into `.format`.